include/mbedtls/ecdsa.h

```diff
@@ -1,10 +1,15 @@
 /**
  * \file ecdsa.h
  *
- * \brief Elliptic curve DSA
+ * \brief Elliptic curve DSA.
+ *
+ * The Elliptic Curve Digital Signature Algorithm (ECSDA) is defined in 
```

Comment:
Blocker: This document defines the use of ECDSA for TLS, but the documentation reads as if it was the original document defining the ECDSA algorithm itself. I think we should look for a better source here, or change the wording.

Reply:
I think it would be more appropriate to cite SEC1 here (which is not the original standard, but it's a good place to look for details - the wording should make that clear), and to prefix the mentioning of RFC 4492 with something like `The use of ECDSA for TLS was defined in ...`.

Reply:
Changed to the following:
ECDSA is defined in Standards for Efficient Cryptography Group (SECG): SEC1 Elliptic Curve Cryptography.
The use of ECDSA for TLS is defined in <em>RFC-4492: Elliptic Curve Cryptography (ECC) Cipher Suites for Transport Layer Security (TLS).